tests/recipes/test_lora_finetune_distributed.py

```diff
@@ -173,6 +175,7 @@ def test_training_state_on_resume(
             resume_from_checkpoint=True \
             metric_logger.filename={log_file} \
             enable_activation_checkpointing=True \
+            enable_activation_offloading=True \
```

Comment:
I enabled it for some tests in this file only. It tests lora and distributed. Ideally, we should have it vs many other parameters, like compile, vision and tiedembeddings. I wont address those in this PR. This should be part of the testing improvement, IMO